src/Microsoft.DocAsCode.Build.Engine/DocumentBuilder.cs

```diff
@@ -215,6 +224,14 @@ public void Build(IList<DocumentBuildParameters> parameters, string outputDirect
                         }
                     }
                 }
+                if (noContentFound)
+                {
+                    Logger.LogWarning($"No content file found. Please make sure the content section of docfx.json is correctly configured.");
+                }
+                else if (emptyContentGroups.Count > 0)
```

Comment:
should this `if` to be embedded in above `if`?

Reply:
Above `if` is for no content totally, this `if` is for content is available overall but some group has no content.

---
In reply to: [185978481](https://github.com/dotnet/docfx/pull/2716#discussion_r185978481) [](ancestors = 185978481)